src/.editorconfig

```diff
@@ -76,3 +76,7 @@ csharp_new_line_before_finally = false
 csharp_new_line_before_members_in_anonymous_types = false
 
 csharp_new_line_before_members_in_object_initializers = false
+
+dotnet_style_qualification_for_field = false:suggestion
+dotnet_style_qualification_for_property = false:none
```

Comment:
I'd rather keep these as they were. We are slowly converting code to C# 7+ so we do want to see suggestion/information for `=>` style so we can fix them as we go.